src/o365/teams/commands/report/teams-report-deviceusageusercounts.spec.ts

```diff
@@ -113,32 +108,12 @@ describe(commands.TEAMS_REPORT_DEVICEUSAGEUSERCOUNTS, () => {
     assert.equal(actual, true);
   });
 
-  it('passes validation on valid \'text\' output', () => {
-    const actual = (command.validate() as CommandValidate)({
-      options: {
-        period: 'D7',
-        output: 'text'
-      }
-    });
-    assert.equal(actual, true);
-  });
-
-  it('passes validation on valid \'json\' output', () => {
-    const actual = (command.validate() as CommandValidate)({
-      options: {
-        period: 'D7',
-        output: 'json'
-      }
-    });
-    assert.equal(actual, true);
-  });
-
-  it('fails validation if specified outputFilePath doesn\'t exist', () => {
+  it('fails validation if specified outputFile doesn\'t exist', () => {
```

Comment:
Shouldn't it be possible to create a new file? Shouldn't this test case be checking the existence of the path rather than a file?

Reply:
@waldekmastykarz Could you please help me with this test case? I am not sure how to stub this here.

Reply:
I didn't mean the contents of the test, just its name. The command shouldn't fail when the target file doesn't exist. It should fail when the target folder doesn't exist. As far as I can see, the current implementation is fine.

Reply:
Ah.. Thank you @waldekmastykarz. I have modified the test case veribage. if required, Please feel free to change and merge.